.nvmrc

```diff
@@ -0,0 +1 @@
+v12
```

Comment:
If this repository is not intended to be `v12`, please change as you see fit and update `@types/node` to follow suite.